db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql

```diff
@@ -1 +1,17 @@
 \echo 'Upgrade Queries not required for transition from $CURRENT_VERSION to $UPGRADE_VERSION'
+
+-- Rollback script for master.app_detail
+ALTER TABLE master.app_detail ALTER COLUMN lang_code SET NOT NULL;
+ALTER TABLE master.app_detail DROP CONSTRAINT pk_appdtl_id;
+
+-- Rollback script for master.biometric_attribute
+ALTER TABLE master.biometric_attribute ALTER COLUMN lang_code SET NOT NULL;
+ALTER TABLE master.biometric_attribute DROP CONSTRAINT pk_bmattr_code;
+
+-- Rollback script for master.module_detail
+ALTER TABLE master.module_detail ALTER COLUMN lang_code SET NOT NULL;
+ALTER TABLE master.module_detail DROP CONSTRAINT pk_moddtl_code;
+
+-- Rollback script for master.template_file_format
+ALTER TABLE master.template_file_format ALTER COLUMN lang_code SET NOT NULL;
+ALTER TABLE master.template_file_format DROP CONSTRAINT pk_tffmt_code;
```

Comment:
We only drop the constraint, we should also add the PK constraint with the lang_code column back @dhanendra06

Reply:
Added the PK constraint with the lang_code column back